Minimum Falling Path

```diff
@@ -0,0 +1,22 @@
+Approach 1 : Brute Force Approach
+Time Complexity :O(n*2)
```

Comment:
Always specify what n stands for. It will answer your question. O(n^2 + n) = O(n^2), where n is the row/col length. Space complexity is correct.